services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -334,6 +334,9 @@ impl NetCdfCfDataProvider {
                         spatial_reference: tree.spatial_reference.into(),
                         measurement: derive_measurement(tail.unit.clone()),
                         no_data_value: None, // we don't want to open the dataset at this point. We should get rid of the result descriptor in the listing in general
+
+                        time: None,
+                        bbox: None,
```

Comment:
Sounds like a task to get these values. We need at least a TODO I guess.

Reply:
66619e218c2104c8577669937023a7354353f531